### PR TITLE
Added gems in Gemfile (versionist, fast_jsonapi, awesome_print)

### PR DESCRIPTION
**Pull request by yashmekhe45**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -56,6 +56,11 @@ gem 'carrierwave-aws'
 gem 'materialize-sass'
 gem 'material_icons'
 
+gem 'versionist'
```
Comment (pramodshinde) [inline Gemfile]: You should add a small note each gem stating why it is added

File: config/database.yml [hunk 2]
```diff
@@ -90,4 +90,4 @@ production:
 
 development:
   <<: *default
-  database: <%= ENV['DATABASE_URL'] %>
```
Comment (pramodshinde) [inline config/database.yml]: Why this is commented ?

File: config/initializers/carrierwave_config.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 
 CarrierWave.configure do |config|
   config.storage    = :aws
-  config.aws_bucket = ENV.fetch('S3_BUCKET_NAME')
+  #config.aws_bucket = ENV.fetch('S3_BUCKET_NAME')
```
Comment (pramodshinde) [inline config/initializers/carrierwave_config.rb]: Why this is commented, @vaibhav8186 check this.

File: dump.rdb [hunk 4]
```diff
@@ -0,0 +1 @@
+REDIS0006�ܳC�Z��V
```
Comment (pramodshinde) [inline dump.rdb]: This file should not be committed

**Comment by pramodshinde** [comment]
PR title should be clear enough to understand, Please correct it.

File: Gemfile (hunk 1 above)
Comment (yashmekhe45) [reply Gemfile]: Sorry, I will make sure to add note for gem in Gemfile.

**Pull request closed without merging**